heidelberg-university-faculty-of-medicine.csl

```diff
@@ -0,0 +1,323 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" page-range-format="expanded" demote-non-dropping-particle="never">
+  <info>
+    <title>Heidelberg University - Faculty of Medicine</title>
```

Comment:
For this type of styles we sometimes include the title in both languages, so that they're easier to find by search, see e.g.:

https://github.com/citation-style-language/styles/blob/master/universita-di-bologna-lettere.csl
`<title>University of Bologna - Liberal Arts College (Università di Bologna - Facoltà di Lettere e Filosofia) (Italian)</title>`

https://github.com/citation-style-language/styles/blob/master/european-journal-of-ultrasound.csl
`<title>European Journal of Ultrasound (Ultraschall in der Medizin)</title>`

https://github.com/citation-style-language/styles/blob/master/veterinary-medicine-austria.csl
`<title>Veterinary Medicine Austria (Wiener Tierärztliche Monatsschrift)</title>`

https://github.com/citation-style-language/styles/blob/master/the-institute-of-electronics-information-and-communication-engineers.csl
`<title>The Institute of Electronics, Information and Communication Engineers (電子情報通信学会)</title>`


Alternative, we could create a dependent for the other locale, and fixed both versions to their respective locales.

I guess the German title would be "Universität Heidelberg - Medizinische Fakultät Heidelberg" (there is a "Medizinische Fakultät Mannheim", and I'm not sure it uses the same style; https://www.umm.uni-heidelberg.de/home/)